packages/orbit-components/transforms/tokens-v16.mjs

```diff
@@ -0,0 +1,87 @@
+import { readFile, writeFile, readdir, stat } from "fs/promises";
+import { join } from "path";
+
+// Define your transformations here
+const transformations = [
+  { from: "rounded-small", to: "rounded-50" },
+  { from: "rounded-t-small", to: "rounded-t-50" },
+  { from: "rounded-tl-small", to: "rounded-tl-50" },
+  { from: "rounded-tr-small", to: "rounded-tr-50" },
+  { from: "rounded-b-small", to: "rounded-b-50" },
+  { from: "rounded-bl-small", to: "rounded-bl-50" },
+  { from: "rounded-br-small", to: "rounded-br-50" },
+  { from: "rounded-l-small", to: "rounded-l-50" },
+  { from: "rounded-r-small", to: "rounded-r-50" },
+  { from: "rounded-x-small", to: "rounded-x-50" },
+  { from: "rounded-y-small", to: "rounded-y-50" },
+  { from: "rounded-normal", to: "rounded-100" },
+  { from: "rounded-t-normal", to: "rounded-t-100" },
+  { from: "rounded-tl-normal", to: "rounded-tl-100" },
+  { from: "rounded-tr-normal", to: "rounded-tr-100" },
+  { from: "rounded-b-normal", to: "rounded-b-100" },
+  { from: "rounded-bl-normal", to: "rounded-bl-100" },
+  { from: "rounded-br-normal", to: "rounded-br-100" },
+  { from: "rounded-l-normal", to: "rounded-l-100" },
+  { from: "rounded-r-normal", to: "rounded-r-100" },
+  { from: "rounded-x-normal", to: "rounded-x-100" },
+  { from: "rounded-y-normal", to: "rounded-y-100" },
+  { from: "rounded-large", to: "rounded-150" },
+  { from: "rounded-t-large", to: "rounded-t-150" },
+  { from: "rounded-tl-large", to: "rounded-tl-150" },
+  { from: "rounded-tr-large", to: "rounded-tr-150" },
+  { from: "rounded-b-large", to: "rounded-b-150" },
+  { from: "rounded-bl-large", to: "rounded-bl-150" },
+  { from: "rounded-br-large", to: "rounded-br-150" },
+  { from: "rounded-l-large", to: "rounded-l-150" },
+  { from: "rounded-r-large", to: "rounded-r-150" },
+  { from: "rounded-x-large", to: "rounded-x-150" },
+  { from: "rounded-y-large", to: "rounded-y-150" },
+  // Add more transformations as needed
+];
+
+async function replaceInFile(filePath) {
+  try {
+    let data = await readFile(filePath, "utf8");
+    let modified = false;
+
+    for (const { from, to } of transformations) {
```

Comment:
Which way do you wanna go?
We can figure these warnings out by rewriting functions (replace `for` loop my functional `.forEach()` (?)) and replacing `console.log`, or we can just put eslint ignore (this approach is used also in other `.mjs` files).

@DSil

Reply:
> replace for loop by functional .forEach()

That is an option, yes.

> we can just put eslint ignore

Yes, on scripts we can use the eslint ignore for console.log

> We also have to polish this script

Since it's a one-off file, we can check the file name and ignore it on purpose.